src/ResourceManager/Automation/Commands.Automation/Cmdlet/ImportAzureAutomationDscNodeConfiguration.cs

```diff
@@ -67,6 +77,7 @@ public override void ExecuteCmdlet()
                     this.AutomationAccountName,
                     this.Path,
                     this.ConfigurationName,
+                    this.IncrementNodeConfigurationBuild.IsPresent,
```

Comment:
So, how does this impact the ShouldConfirm prompt that your Force parameter is bypassing?